lib/msf/ui/console/module_argument_parsing.rb

```diff
@@ -53,7 +53,7 @@ def parse_run_opts(args, action: nil)
       end
     end
 
-    parse_opts(@@module_opts_with_action_support, args, help_cmd: help_cmd)
+    parse_opts(@@module_opts_with_action_support, args, help_cmd: help_cmd, action: action)
```

Comment:
Landed in master already: https://github.com/rapid7/metasploit-framework/pull/15737